ios/ImagePickerManager.m

```diff
@@ -85,13 +85,22 @@ - (void) showPickerViewController:(UIViewController *)picker
 - (void)imagePickerController:(UIImagePickerController *)picker didFinishPickingMediaWithInfo:(NSDictionary<NSString *,id> *)info
 {
     dispatch_block_t dismissCompletionBlock = ^{
+        NSMutableArray<NSDictionary *> *assets = [[NSMutableArray alloc] initWithCapacity:1];
+
         if ([info[UIImagePickerControllerMediaType] isEqualToString:(NSString *) kUTTypeImage]) {
-            [self onImageObtained:[ImagePickerManager getUIImageFromInfo:info]
-                             data:[NSData dataWithContentsOfURL:[ImagePickerManager getNSURLFromInfo:info]]];
```

Comment:
`getNSURLFromInfo` returns a nil NSURL here because the info key it uses doesnt exist. See https://imgur.com/a/FoQAN5z for some debugger output on these lines.

Without refactoring the logic too much, i just pass in `nil` for the data argument on line 92.